joomla/joomla-cms
# Fix error (stdClass|array) to string conversion in article version history

Display (Array | Object) attribs parameters as String in Article Version History

Pull Request for Issue #38261 .

### Summary of Changes

Article attribs can contain Array or Object properties, they are not properly displayed in backend Article Version History

### Testing Instructions

Install Astroid Framework that sets some article attribs parameters as arrays.

- Create an article and save it.
- Click button "Versions".
- Click a date link to see the overview/preview. Scroll down.

### Actual result BEFORE applying this Pull Request

![178563271-98fdcdee-8b38-49f6-b033-4cb760dddb47](https://user-images.githubusercontent.com/3204103/178951422-5dba8d23-457f-45d4-87a9-387b1b5d8c62.png)
![178563349-54fff5ee-f910-46ef-8ac1-bf1422878cd2](https://user-images.githubusercontent.com/3204103/178951588-df50afc0-f14b-4abc-91d1-62a112dab990.png)

### Expected result AFTER applying this Pull Request

![Screen Shot 2022-07-14 at 16 32 38](https://user-images.githubusercontent.com/3204103/178951709-b8ab8502-362c-4208-870e-2cf3f9ec06d2.png)

(Array | Object) parameters will display in JSON String format
This pull request will solve issue #38261 

### Documentation Changes Required

No need
